doc/gs_single_cell.rst

```diff
@@ -0,0 +1,120 @@
+.. _gs_single_cell:
+
+A single cell model
+================================
+
+Building and testing detailed models of individual cells, then optimizing their parameters is usually the first step in building models with multi-compartment cells. Arbor supports a *single cell model* workflow for this purpose, which is a good way to introduce Arbor's cell modeling concepts and approach.
+
+This guide will walk through a series of single cell models of increasing complexity. Links are provide to separate documentation that covers relevant topics in more detail.
+
+In an interactive Python interpreter, you can use ``help()`` on any class or function to obtain some documentation. E.g.: ``help(arbor.gap_junction_connection)`` will print :class:`this<arbor._arbor.gap_junction_connection>`.
+
+.. _single_soma:
+
+Single compartment cell with HH dynamics
+----------------------------------------------------
+
+The most trivial representation of a cell in Arbor is to model the entire cell as a sphere.
+The following model shows the steps required to construct a model of a spherical cell with
+radius 3 μm, Hodgkin–Huxley dynamics and a current clamp stimulus, then run the model for
+30 ms.
+
+The first step is to construct the cell. In Arbor, the abstract representation used to define
+a cell with branching "cable" morphology is a ``cable_cell``, which holds a description
+of the cell's morphology, named regions and locations on the morphology, and descriptions of
+ion channels, synapses, spike detectors and electrical properties.
+
+Our "single-compartment HH cell" has a simple morphology and dynamics, so the steps to
+create the ``cable_cell`` that represents it are as follows:
+
+.. code-block:: python
+
+    import arbor
+
+    # (1) Create a morphology with a single (cylindrical) segment of length=diameter=6 μm
+    tree = arbor.segment_tree()
+    tree.append(arbor.mnpos, arbor.mpoint(-3, 0, 0, 3), arbor.mpoint(3, 0, 0, 3), tag=1)
+
+    # (2) Define the soma and its center
+    labels = arbor.label_dict({'soma':   '(tag 1)',
+                               'center': '(location 0 0.5)'})
+
+    # (3) Create cell and set properties
+    cell = arbor.cable_cell(tree, labels)
+    cell.set_properties(Vm=-40)
+    cell.paint('soma', 'hh')
+    cell.place('center', arbor.iclamp( 10, 2, 0.8))
+    cell.place('center', arbor.spike_detector(-10))
+
```

Comment:
Even in an arbitrary label?

Reply:
I think _because_ it is arbitrary :) If it were part of the API, it would be a different story. (I am not looking forward to fights over API spelling, to the point that I want to avoid having anything involving centres or colours in the interface…)

Reply:
This reflects the same choice as in the python example, so then let's keep things consistent :)